x/slashing/spec/02_state.md

```diff
@@ -44,31 +52,23 @@ The information stored for tracking validator liveness is as follows:
 // ValidatorSigningInfo defines a validator's signing info for monitoring their
 // liveness activity.
 message ValidatorSigningInfo {
+  option (gogoproto.equal)            = true;
+  option (gogoproto.goproto_stringer) = false;
```

Comment:
i propose to use

```
+++ https://github.com/cosmos/cosmos-sdk/blob/v0.40.0/proto/cosmos/slashing/v1beta1/slashing.proto#L11-L30
```

so that it's easier to maintain. `+++` will paste the code snippet in the docs.